eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -22,15 +22,15 @@ foreach ($branch in $branches)
     $branchName = $branch.Replace("refs/heads/","")
     $head = "${RepoOwner}/${RepoName}:${branchName}"
     LogDebug "Operating on branch [ $branchName ]"
-    $pullRequests = Get-GitHubPullRequests -RepoOwner $RepoOwner -RepoName $RepoName -head $head
+    $pullRequests = Get-GitHubPullRequests -RepoOwner $RepoOwner -RepoName $RepoName -State "all" -head $head
```

Comment:
What is the default state if you don't pass one? Can you just pass open as the state?

Reply:
The default state is open. Am trying to get all the pull request then only delete the branch if there is no open pull request among the returned result. I don't want to delete a branch that was just created but the pull request has not ye been created off it.